resources/views/master.blade.php

```diff
@@ -161,7 +161,7 @@
                                 </li> -->
                                 <li>
                                     <i class="fa fa-envelope gradient"></i>
-                                    <a href="mailto:medhadigitalsolutions@gmail.com" target="_blank" class="text-white">medhadigitalsolutions@gmail.com</a>
+                                    <a href="mailto:team@medha.digital" target="_blank" class="text-white">medhadigitalsolutions@gmail.com</a>
```

Comment:
Change both visible and mailto: text
now it will show the gmail id in view but send mail to team@medha.digital while clicking on it.